.github/workflows/README.md

```diff
@@ -0,0 +1,89 @@
+
+Following the installation of system dependencies, R, and package dependencies, each workflow does something different.
+
+### [`check-templates.yml`](check-templates.yml)
+
+This workflow checks for issues within template scripts.  For example, in admiral package there are several template scripts with admiral-based functions showing how to build certain ADaM datasets.   As we update the admiral functions, we want to make sure these template scripts execute appropriately.  Functions in the template scripts that are deprecated or used inappropriately will cause this workflow to fail. 
+
+### [`code-coverage.yml`](code-coverage.yml)
+
+This workflow measures code coverage for unit tests and reports the code coverage as a percentage of the _total number of lines covered by unit tests_ vs. the _total number of lines in the codebase_.
+
+The [`covr`][covr] R package is used to calculate the coverage.
+
+Report summaries and badges for coverage are generated using a series of other GitHub Actions.
+
+### [`links.yml`](links.yml)
+
+This workflow checks whether URLs embedded in code and documentation are valid. Invalid URLs reults in workflow failures. This workflow uses [`lychee`][lychee] to detect broken links.   Occasionally this check will detect false positives of urls that look like urls.  To remedy, please add this false positive to the `.lycheeignore` file.   
+
+### [`lintr.yml`](lintr.yml)
+
+Static code analysis is performed by this workflow, which in turn uses the [`lintr`][lintr] R package.
+
+Any [`.lintr`](../../.lintr) configurations in the repository will be by this workflow.
+
+### [`pkgdown.yml`](pkgdown.yml)
+
+Documentation for the R package is generated via this workflow. This workflow uses the [`pkgdown`](pkgdown) framework to generate documentation in HTML, and the HTML pages are deployed to the `gh-pages` branch.
+
+Moreover, an additional `Versions` dropdown is generated via the [][multi-version-docs] GitHub Action, so that an end user can view multiple versions of the documentation for the package.
+
+### [`r-cmd-check.yml`](r-cmd-check.yml)
+
+This workflow performs `R CND check` for the package. Failed workflows are typically indicative of problems encountered during the check, and therefore an indication that the package does not meet quality standards.
+
+### [`r-pkg-validation.yml`](r-pkg-validation.yml)
+
+When a new release of the package is made, this workflow executes to create a validation report via [theValidatoR][validation]. The PDF report is then attached to the release within GitHub.
+
+### [`readme-render.yml`](readme-render.yml)
+
+If your codebase uses a `README.Rmd` file], then this workflow will automatically render a `README.md` and commit it to your branch.
```

Comment:
very cool @dinakar29 - all reads well and makes sense to me. the only minor thing i spotted is a seemingly stray `]` here

Reply:
Good find. I'll remove this.